gitea-dev/deployment.yaml

```
@@ -0,0 +1,48 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: gitea-dev
```

この辺の名前から `-dev` を削除して欲しいです。（本番環境との設定との差異は内容だけにしたいため）

```suggestion
    app: gitea
```